Formula/bombadillo.rb

```diff
@@ -0,0 +1,23 @@
+class Bombadillo < Formula
+  desc "Non-web browser, designed for a growing list of protocols"
+  homepage "http://bombadillo.colorfield.space"
+  url "https://tildegit.org/sloum/bombadillo/archive/2.2.0.tar.gz"
+  sha256 "037a5e70b02fcf34b7d01f710111576b66faf117bceb000b527bff13ce939f3c"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", "-o", bin/"bombadillo"
+    man1.install "bombadillo.1"
+    prefix.install_metafiles
```

Comment:
You can leverage existing `Makefile`:
```suggestion
    system "make", "build", "GOCMD=#{Formula["go"].opt_bin/"go"}"
    system "make", "install", "PREFIX=#{prefix}"
```

Reply:
Thanks, this is a great suggestion. I changed the formula to use `make install` and it doesn't seem to need a separate `make build` step.

Reply:
> Thanks, this is a great suggestion. I changed the formula to use `make install` and it doesn't seem to need a separate `make build` step.

You are right 👍